content/en/Integrations/IntegrationBuilder/How to Guides/github.md

```diff
@@ -0,0 +1,164 @@
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/11_use-recipe.png" alt="Use GitHub recipe" %}}
+
+3. Select the folder you created and click "Copy and save."
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/12_copy-recipe.png" alt="Copy GitHub recipe" %}}
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/13_copy-and-save.png" alt="Copy and save" %}}
+
+### Customize the recipe
+
+You will need to customize the recipe to use your GitHub organization and
+repository. You can optionally customize the content of the GitHub issues that
+the recipe creates.
+
+1. Click the "Customize recipe" button to open the recipe editor.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/14_customize-recipe-start.png" alt="Start customizing" %}}
+
+2. Click the link for step 17 in the error message at the top of the editor.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/15_customize-error.png" alt="Link for step 17" %}}
+
+3. Select or input your GitHub Organization and Repository name required fields.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/16_customize-recipe.png" alt="Input required fields" %}}
+
+  - **Optional**: Customize the content of the GitHub issues that the recipe creates.
+    - You can use datapills to populate the fields with dynamic values.
+    - You can type into the input fields to set static values.
+
+    {{% image src="/integrations/integration_builder/how_to_guides/github/24_customizable-fields.png" alt="Customizable fields" %}}
+
+    - Available fields:
+      - Issue title
+      - Body
+      - Assignees
+      - Labels (optional field, see below)
+      - Milestone ID (optional field, see below)
+
+      {{% image src="/integrations/integration_builder/how_to_guides/github/25_optional-fields.png" alt="Optional fields" %}}
+
+      {{% image src="/integrations/integration_builder/how_to_guides/github/26_select-optional-fields.png" alt="Select optional fields" %}}
+
+4. Click the "Save" button to save your changes.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/17_save-recipe.png" alt="Save recipe" %}}
+
+5. Click the "Exit" button to close the recipe editor.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/18_exit-recipe-editor.png" alt="Exit recipe editor" %}}
+
+6. By default, the recipe will be in an inactive state. To start pushing your
+findings to GitHub issues, click the "Start recipe" button.
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/19_start-recipe.png" alt="Start GitHub recipe" %}}
+
+{{% image src="/integrations/integration_builder/how_to_guides/github/20_recipe-running.png" alt="Recipe running" %}}
+
+## Test the recipe
+
+To test the recipe with pentest findings, follow the guide for
```

Comment:
I might have missed it, but I couldn't find instructions on where/how to change the pentest for which issues should be created. The "Edit the recipe" section comes close to that, but doesn't explicitly mention that the pentest can be selected there.

Reply:
I didn't even think about this. @grayson-cobalt can correct me if I'm wrong, but I believe it will fire for all pentests/findings within the org by default.

I just looked at it more carefully and the trigger can be set to fire for either a specific asset or a pentest. There are also trigger conditions, but that already links out to external documentation so I'm not sure I need to add anything about that.

I'll add a section for how to update the trigger.

Reply:
Ah ok, I wasn't aware the trigger worked for the whole org as well. Then there's probably no need to mention it, maybe just a note.